src/js/smooch.jsx

```diff
@@ -196,6 +195,40 @@ export class Smooch {
         });
     }
 
+    monitorUrlChanges() {
+        const onhashchange = window.onhashchange;
+        const pushState = window.history.pushState;
+        const replaceState = window.history.replaceState;
+
+        window.onhashchange = function() {
```

Comment:
Results in this error: 
uncaught TypeError: Failed to execute 'pushState' on 'History': 2 arguments required, but only 0 present

So the arguments are empty or something...